README.md

```diff
@@ -95,4 +95,9 @@ $ tox
 This will run the tests with several versions of Python 3, measure coverage,
 and run flake8 for code linting.
 
+## Run in Docker
+```
+docker run tufin/operator-courier operator-courier
```

Comment:
I would prefer, based on your latest change, that the README did not point to a particular docker image. Instead it should describe how to build the docker image, i.e.:

```
docker build Dockerfile -t $TAG
docker run $TAG operator-courier
```

Reply:
done